<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWidget</class>
 <widget class="QWidget" name="settingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize</string>
  </property>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (9~24):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width (9~30):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_h">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>9</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_w">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>9</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_m">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>31</height>
    </size>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>668</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_m">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mines (10~80):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
